build/run-in-docker.sh

```diff
@@ -70,6 +70,7 @@ echo "PLATFORM_FLAG=$PLATFORM_FLAG"
 
 if [[ "$DOCKER_IN_DOCKER_ENABLED" == "true" ]]; then
   echo "..reached DIND check TRUE block, inside run-in-docker.sh. Checking environ"
+  go get github.com/onsi/ginkgo/ginkgo@v1.16.4
```

Comment:
maybe we should use `go install `

Reply:
I also think, we should use go install but I stuck with what is consistent in existing code. We need to bump ginkgo, go and maybe bunch of other stuff too